### PR TITLE
fixed marcus not detecting 'x=__import__'

### PR DESCRIPTION
**Pull request by ethanlchristensen**
deez!

### CODE REVIEW

File: BOT/_utils/lifegen.py
Comment (ethanlchristensen) [inline BOT/_utils/lifegen.py:1]: Awesome work, ethan!

**Comment by ethanlchristensen** [comment]
don't know if that most recent commit will work

**Review by Mr-Oregano — APPROVED** [review]
Approved without looking at the code! 😎

**Review by nulzo — APPROVED** [review]
nice work :-)

**Pull request merged**